.github/CODEOWNERS

```diff
@@ -966,6 +966,15 @@
 # ServiceOwners:                                                   @jamesfan1 @bangbingsyb @spencerofwiti
 
 
+# PRLabel: %ARM
+/sdk/resources/Azure.ResourceManager.*/                            @anamikapan
+
+# ServiceLabel: %ARM %Mgmt
+# ServiceOwners:                                                   @anamikapan
+
+# PRLabel: %Recovery Services Backup
+/sdk/recoveryservices-backup/Azure.ResourceManager.*/              @Daya-Patil
+
```

Comment:
```suggestion
```

This library is not related to your changes.

Reply:
@jsquire arthur suggested this change for some reason as it was showing some errors in codeowners file and carry forward those changes. I can remove it

Reply:
Adding a PR entry where one didn't exist won't resolve a linter error.   If we're seeing errors pop up, it likely means that someone left their team or the company and we've got to scrub data.   I don't see an error to fix at the moment.

Reply:
That said, if Arthur says this person should be on the hook for reviews, then I'm good leaving it.